### PR TITLE
Added 3 Binary Pages

### PR DESCRIPTION
**Pull request by georgeherby**
3 binary pages added:
- Package Copack Small
- Import
- Copacked

Pages route to their correct pages based on their true/false value.

### CODE REVIEW

File: app/sdil/controllers/CopackedController.scala [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        validFormData => {
+          if (validFormData){
+            Redirect(routes.LitreageController.show("copackedVolume"))
+          }
+          else{
+            Redirect(routes.ImportController.display())
+          }
+        })
+    }
+  private lazy val booleanMapping: Mapping[Boolean] =
```
Comment (mattrobertsky) [inline app/sdil/controllers/CopackedController.scala]: This is a dupe of the one in the SDILController. Both should probably be moved either to the controller package or the forms package.

File: app/sdil/controllers/ImportController.scala [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        validFormData => {
+          if (validFormData){
+            Redirect(routes.LitreageController.show("importVolume"))
+          }
+          else{
+            Redirect(routes.StartDateController.show())
+          }
+        })
+    }
+  private lazy val booleanMapping: Mapping[Boolean] =
```
Comment (mattrobertsky) [inline app/sdil/controllers/ImportController.scala]: ditto

File: app/sdil/controllers/PackageCopackSmallController.scala [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        validFormData => {
+          if (validFormData){
+            Redirect(routes.LitreageController.show("packageCopackSmallVol"))
+          }
+          else{
+            Redirect(routes.CopackedController.display())
+          }
+        })
+    }
+  private lazy val booleanMapping: Mapping[Boolean] =
```
Comment (mattrobertsky) [inline app/sdil/controllers/PackageCopackSmallController.scala]: ditto

File: app/sdil/controllers/CopackedController.scala [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+ */
+
+package sdil.controllers
+
+import javax.inject.Inject
+
+import play.api.data.Forms.{boolean, optional, single}
+import play.api.data.{Form, Mapping}
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent}
```
Comment (mattrobertsky) [inline app/sdil/controllers/CopackedController.scala]: AnyContent is unused.

File: app/sdil/controllers/ImportController.scala [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+ */
+
+package sdil.controllers
+
+import javax.inject.Inject
+
+import play.api.data.Forms.{boolean, optional, single}
+import play.api.data.{Form, Mapping}
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent}
```
Comment (mattrobertsky) [inline app/sdil/controllers/ImportController.scala]: AnyContent is unused.

File: app/sdil/controllers/PackageCopackSmallController.scala [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sdil.controllers
+
+import java.util.UUID
```
Comment (mattrobertsky) [inline app/sdil/controllers/PackageCopackSmallController.scala]: unused UUID

File: app/sdil/controllers/PackageCopackSmallController.scala [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+import javax.inject.Inject
+
+import play.api.data.{Form, Mapping}
+import play.api.data.Forms.{boolean, mapping, optional, single, text}
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent}
```
Comment (mattrobertsky) [inline app/sdil/controllers/PackageCopackSmallController.scala]: unused AnyContent

File: test/sdil/controllers/CopackedControllerSpec.scala [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+        status(res) mustBe SEE_OTHER
+        redirectLocation(res).value mustBe routes.ImportController.display().url
+      }
+    }
+  }
+  lazy val testController = new CopackedController(messagesApi) {
+    override val cache: SessionCache = mockCache
+  }
+
+  lazy val mockCache = {
```
Comment (mattrobertsky) [inline test/sdil/controllers/CopackedControllerSpec.scala]: Move this (and instances below) to controllerHelpers.

**Comment by georgeherby** [comment]
@mattrobertsky changes made as per the comments above

**Pull request merged**